websites/T/TruckStopRadio/presence.ts

```diff
@@ -1,54 +1,183 @@
 const presence = new Presence({
-		clientId: "854817731161489449"
+		clientId: "639107568672702484"
 	}),
-	browsingTimestamp = Math.floor(Date.now() / 1000),
-	dj = document.querySelector("#presenter-name") as HTMLElement;
+	getStrings = async () => {
```

Comment:
Can you make a proper function?

Reply:
Doesn't matter and a lot of presences have it like this. Actually most of them do...

Reply:
They should use a function though?